driver/src/price_finding/optimization_price_finder.rs

```diff
@@ -223,7 +212,7 @@ impl PriceFinding for OptimisationPriceFinder {
         state: &models::AccountState,
     ) -> Result<models::Solution> {
         let input = solver_input::Input {
-            tokens: serialize_tokens(&orders, &self.token_data),
+            tokens: self.price_oracle.get_token_prices(&orders),
```

Comment:
Do we know how long this roughly takes? Given that it's on the time critical path for now it would be good to have an idea and maybe capture somewhere a follow up task to do the price estimation just before a batch can be solved.

Reply:
I added a timer to the `KrakenClient` test, which tries to retrieve prices for 10 tokens and this is what I got:

```
Total elapsed time: 614.848532ms
```

For reference, the test is here: https://github.com/gnosis/dex-services/blob/09746e7382cf77bbf4df9ca6e6167715ac0ba0cb/driver/src/price_estimation/kraken.rs#L171-L206

Reply:
We might be able to cut down the time a bit if we get the asset and asset pairs together, and maybe check if there is any HTTP caching we can do.

Reply:
Created  #583

Reply:
Ok, well since it's less than 1s it's negligible